internal/compiler/builtins.slint

```diff
@@ -415,7 +415,21 @@ PropertyAnimation := _ {
 
 export struct StandardListViewItem := {
     //-name:slint::private_api::StandardListViewItem
-    text: string
+    text: string,
+    editable: bool,
+}
+
+export struct TableColumn := {
+    //-name:slint::private_api::TableColumn
+    title: string,
+    min-width: length,
+    horizontal-stretch: float,
+    sort-by-ascending: bool,
+    sort-by-descending: bool,
+}
+
+export struct StandardTableRow := {
+    items: [StandardListViewItem]
```

Comment:
I wonder if we really need a `StandardTableRow` or if it can just be a `[StandardListViewItem]`.
Is there any other field that `items` that could be added in the future?

Reply:
I think you're right. At the moment an extra type is not necessary.